src/main/java/mate/academy/onlinebookstore01/controller/BookController.java

```diff
@@ -0,0 +1,35 @@
+package mate.academy.onlinebookstore01.controller;
+
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import mate.academy.onlinebookstore01.dto.BookDto;
+import mate.academy.onlinebookstore01.dto.CreateBookRequestDto;
+import mate.academy.onlinebookstore01.service.BookService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping(value = "/api/books")
+public class BookController {
+    private final BookService bookService;
+
+    @GetMapping
+    public List<BookDto> getAll() {
+        return bookService.findAll();
+    }
+
+    @GetMapping("/api/books/{id}")

```

Comment:
You already defined `/api/books` part in `@RequestMapping` annotation.
Did you test your solution?
```suggestion
    @GetMapping("/{id}")
```